client/src/pages/CreateAction.js

```diff
@@ -1,25 +1,154 @@
 import "./CreateAction.css";
 import Card from "../components/Card";
-import HeaderGoForward from "../components/Card_components/HeaderGoForward";
+import HeaderActionGoForward from "../components/Card_components/HeaderActionGoForward";
 import SpeechInput from "../components/SpeechInput";
-import { useParams } from "react-router";
+import { useParams, useHistory } from "react-router";
+import { useState, useEffect } from "react";
 import FooterSubmit from "../components/Card_components/FooterSubmit";
 
 export default function CreateCase() {
+  /** VARIABLE SET UP */
+  const reminderArray = JSON.parse(localStorage.getItem("newEntry"));
   const { number } = useParams();
+  const indexNumber = Number(number);
+  const history = useHistory();
+  const [verbInput, setVerbInput] = useState(
+    reminderArray.tasks[indexNumber - 1].verb
+  );
+  const [actionInput, setActionInput] = useState(
+    reminderArray.tasks[indexNumber - 1].action
+  );
+  const [withInput, setWithInput] = useState(
+    reminderArray.tasks[indexNumber - 1].with
+  );
```

Comment:
It is important to use 4 states for every data? Could you just use one state?
```js
const [inputs, setInputs] = useState({
    verb: reminderArray.tasks[indexNumber - 1].verb,
    action: reminderArray.tasks[indexNumber - 1].action,
    with: reminderArray.tasks[indexNumber - 1].with
    })
```

And then changing this accordingly.